src/modules/layout/components/HeaderActionBar.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
```

Comment:
I think the naming of HeaderActionBar and HeaderMenuBar is not quite clear, maybe we should think of different names

Reply:
yeah kind of true :)

Reply:
I think HeaderActionBar is good, for HeaderMenuBar I would maybe suggest HeaderNavigationBar or something like that

Reply:
Done.